backend/audio_processor.py

```
@@ -0,0 +1,299 @@
+            self.tts_models["en"] = TTS(
+                model_name="tts_models/en/ljspeech/tacotron2-DDC",
+                gpu=self.device == "cuda"
+            )
+            
+            # Japanese TTS
+            try:
+                self.tts_models["ja"] = TTS(
+                    model_name="tts_models/ja/kokoro/tacotron2-DDC",
+                    gpu=self.device == "cuda"
+                )
+            except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)